deployments/helm-chart/templates/controller-service.yaml

```diff
@@ -14,6 +14,11 @@ metadata:
 {{ toYaml .Values.controller.service.annotations | indent 4 }}
 {{- end }}
 spec:
+{{- if .Values.controller.service.clusterIP }}
+  {{- if not (eq .Values.controller.service.clusterIP "") }}
+  clusterIP: {{ .Values.controller.service.clusterIP }}
+  {{- end }}
+{{- end }}
```

Comment:
I think this can be changed like this:
```suggestion
{{- if .Values.controller.service.clusterIP }}
  clusterIP: {{ .Values.controller.service.clusterIP }}
{{- end }}
```
unless I'm missing something?

Reply:
Hi, it's ok from my point of view, if you leave it blank Kubernetes will assign one ip for you.
Thanks
Stefano

Reply:
ok, thanks

Reply:
@EutiziStefano would you like to make the suggested change?